test/tests/amqp/test_amqp_racknode.py

```diff
@@ -0,0 +1,166 @@
+'''
+Copyright 2017 Dell Inc. or its subsidiaries.  All Rights Reserved.
+
+Author(s):
+Norton Luo
+
+'''
+import json
+from time import sleep
+import os
+import sys
+import json
+import time
+import string
+from datetime import *
+import time
+import subprocess
+# set path to common libraries
+sys.path.append(subprocess.check_output("git rev-parse --show-toplevel", shell=True).rstrip("\n") + "/test/common")
+import fit_common
+import test_api_utils
+import fit_amqp
+
+amqp_message_received = False
+routingkey = ""
+amqpbody = ""
+
+
+def get_node_list_by_type(type):

```

Comment:
I believe there are already common methods for this. I believe in common/test_api_utils.py